erdpy/config.py

```diff
@@ -1,9 +1,12 @@
+from itertools import chain
+from logging import Logger
 import os.path
-from typing import Any, Dict
+from typing import Any, Dict, List
 
 from erdpy import errors, utils
 
 ROOT_FOLDER_NAME = "elrondsdk"
+LOCAL_CONFIG_PATH = os.path.join(os.getcwd(), "erdpy.json")
 CONFIG_PATH = os.path.expanduser("~/elrondsdk/erdpy.json")
```

Comment:
We should rename `CONFIG_PATH` to `GLOBAL_CONFIG_PATH` now that we have `LOCAL_CONFIG_PATH`.

Reply:
Done.